Library/Homebrew/sandbox.rb

```diff
@@ -95,6 +95,10 @@ def exec(*args)
       -k Sender sandboxd
     ]
     logs = Utils.popen_read("syslog", *syslog_args)
+
+    # These messages are confusing and non-fatal, so don't report them.
+    logs = logs.gsub(/^.*Python(\d+) deny file-write.*pyc$/, "").strip
```

Comment:
It'd have to be a multiline regex, it turns out; I don't know why I thought `strip` would fix this. ☕☕☕ Tried an approach with `.lines` instead; let me know how you like it.